docs/_interface/sync-modes.md

```diff
@@ -0,0 +1,118 @@
+chain much faster when they are new or have been offline for a while. The trade-off is that light nodes rely heavily 
+on data served by altruistic full nodes. A light client can be used to query data from Ethereum and submit transactions, 
+acting as a locally-hosted Ethereum wallet. However, because they don't keep local copies of the Ethereum state, light 
+nodes can't validate blocks in the same way as full nodes - they have to trust that the data they receive is honest. 
+To start a node in light mode, pass `--syncmode light`. Be aware that full nodes serving light data are relative scarce 
+so light nodes can struggle to find peers.
+
+Read more about light nodes on our [LES page](/docs/interface/les.md).
+
+## Consensus layer syncing
+
+At the transition to proof-of-stake, all consensus logic and block propagation is handed over to consensus clients. 
+This means that syncing the blockchain is a process shared between the consensus and execution clients. Blocks are 
+downloaded by the consensus client and verified by the execution client. There are two ways to sync a consensus 
+client: optimistic sync and checkpoint sync. 
+
+### Optimistic sync
+
+Optimistic sync downloads blocks before the execution client has validated them. This means the execution client can 
+constantly be fed with up-to-date states to snap-sync to. Assuming the blocks are valid, eventually the sync catches 
+up to the optimistically downloaded head and the blockchain is in sync. From this point, verification is done 
+block-by-block (i.e. the sync mode switches to `full`). In optimistic sync the node assumes the data it receives 
+from its peers is correct during the downloading phase but then retroactively verifies each downloaded block. Nodes 
+are not allowed to attest or propose blocks while they are still 'optimistic' because they can't yet guarantee their 
+view of the ehad of the chain is correct.
+
+Read more in the [optimistic sync specs](https://github.com/ethereum/consensus-specs/blob/dev/sync/optimistic.md).
+
+### Checkpoint sync
+
+Alternatively, the consensus client can grab a checkpoint from a trusted source which provides a target state to sync 
+up to, before switching to full sync and verifying each block in turn. In this mode, the node trusts that the checkpoint 
+is correct. There are many possible sources for this checkpoint - the gold standard would be to get it out-of-band 
+from another trusted friend, but it could also come from block explorers or public APIs/web apps.
+
+**Note** it is not currently possible to use a Geth light node as an execution client on proof-of-stake Ethereum.
+
+## Summary
+
+There are several ways to sync a Geth node. The default is to use snap sync to create a full node. This verifies all 
+blocks starting at a recent checkpoint. A trust-minimized alternative is full-sync, which verifies every block since genesis. 
```

Comment:
recent checkpoint feels weird. Actually there is no checkpoint notion in snap sync. We just basically choose a recent block as the syncing target. The target won't be chosen as the current head since it might be reorged. Instead we usually choose `current.head - 64` which is new enough but still has enough block confirmations on top.